src/Faker/Provider/vi_VN/Internet.php

```diff
@@ -4,5 +4,346 @@
 
 class Internet extends \Faker\Provider\Internet
 {
-    protected static $tld = array('com', 'com', 'com', 'com', 'com', 'com', 'biz', 'info', 'net', 'org', 'vn', 'com.vn', 'biz.vn', 'edu.vn', 'gov.vn', 'net.vn', 'org.vn', 'int.vn', 'ac.vn', 'pro.vn', 'info.vn', 'health.vn', 'name.vn', 'mil.vn');
+    protected static $tld = array(
+        'com',
+        'biz',
+        'info',
+        'net',
+        'org',
+        'vn',
+        'name',
+        'pro',
+        'aero',
+        'asia',
+        'cat',
+        'coop',
+        'edu',
+        'gov',
+        'int',
+        'jobs',
+        'mil',
+        'mobi',
+        'museum',
+        'tel',
+        'travel',
+        
+        'ac',
```

Comment:
Please group this alphabetically, e.g. 80 characters on a line so the `git diff` is still readable. See area codes beneath

Reply:
I'm not sure if you want to provide all ccTLD? Why not just stick to Vietnamese and gTLD?

Reply:
A very good idea. I will try to modify it in near future!